### PR TITLE
Trigger workflow cleanup

### PR DESCRIPTION
**Pull request by saraycp**
After rolling out the `trigger_workflow` feature more than 4 months ago, it's time to clean up the code to remove all the feature flag checks.

**How to test?**

Make sure your user is [out of beta](http://localhost:3000/my/beta_features) and try to create/edit/update a workflow token [from the UI](http://localhost:3000/my/tokens) and also trigger one.

### CODE REVIEW

**Comment by dmarcoux** [comment]
There's a typo in the third commit, **triggle**

File: src/api/app/policies/token/rss_policy.rb
Comment (saraycp) [inline src/api/app/policies/token/rss_policy.rb:3]: `Token::RssPolicy#create?` is the same as `TokenPolicy#create?` after the changes in [the second one](https://github.com/openSUSE/open-build-service/pull/13810/commits/eb2754812ed1405de2b0f64a2dc68aa0b8c32701#diff-f156eef00f70113b2298c238401790daaed323194494afc4e29093d3355774f6L34).

**Comment by saraycp** [comment]
I could squash some commits before merging...

**Review by rubhanazeem — APPROVED** [review]

**Review by danidoni — APPROVED** [review]

**Comment by saraycp** [comment]
I squashed some of the commits. I'll merge once the checks pass again.

**Pull request merged**
